cypress/integration/frontsite/details.test.spec.js

```diff
@@ -0,0 +1,104 @@
+//https://127.0.0.1:3000/search?f=doctype%3ABOOK
+
+describe('frontsite literature details', () => {
+  const goToSearchDocument = () => {
+    // Books + available for loan
+    cy.visit(
+      '/search?f=doctype%3ABOOK&f=availability%3Aavailable%20for%20loan'
+    );
+  };
+
+  it('can view document details and request loan if available', () => {
+    cy.server();
+    cy.route('GET', '/api/locations/*').as('location');
```

Comment:
`L12`: tell the framework to listen for `xhr` requests
`L13`: name this specific route pattern as `location`, which can later be used by calling `@location`. It's not possible to register the route on the fly, because of the way the framework works. Perhaps we can have a global store for all the patterns we are interested in listening to.